glean-core/csharp/GleanTests/DispatchersTests.cs

```diff
@@ -66,23 +43,25 @@ public void TestTaskQueueing()
             Dispatchers.TestingMode = true;
             Dispatchers.QueueInitialTasks = true;
 
+            List<Task> tasks = new List<Task>();
+
             // Add 3 tasks to the queue, each one incrementing threadCanary to
             // indicate the task has executed.
             for (int i = 0; i < 3; ++i)
             {
-                Dispatchers.LaunchAPI(() =>
+                tasks.Add(Dispatchers.LaunchAPI(() =>
                 {
                     threadCanary += 1;
-                });
-            }
+                }));
+            }       
```

Comment:
```suggestion
            }
```

I don't really like the linter not complaining about this :\

Reply:
I don't really like VS putting it there either, they aren't exactly visible in the editor.